interface/main/holidays/import_holidays.php

```diff
@@ -85,6 +91,9 @@
 
     //Upload and save the csv
     $saved = $holidays_controller->create_holiday_event();
+    if (!$saved && empty($error_message)) {
+        $error_message = xlt('Operation Failed');
```

Comment:
`xlt` is appropriate to prevent the possibility of xss in a translated string; however, it's probably more consistent to use `xl` here, because the only time I see `$error_message` being used in frontend output, it's wrapped in `text()`, which would also prevent the xss.